disperser/dataapi/server.go

```diff
@@ -244,6 +250,7 @@ func (s *server) Start() error {
 		{
 			feed.GET("/blobs", s.FetchBlobsHandler)
 			feed.GET("/blobs/:blob_key", s.FetchBlobHandler)
+			feed.GET("/blobs/:accountId", s.FetchBlobCountByAccountIdHandler)
```

Comment:
Can we make it more explicit that we're returning a count in the API name? Maybe `/blobs/:accountId/count`?